app/assets/javascripts/govuk_publishing_components/components/checkboxes.js

```diff
@@ -1,75 +1,74 @@
-// This component relies on JavaScript from GOV.UK Frontend
-//= require govuk-frontend/components/checkboxes/checkboxes.js
-window.GOVUK = window.GOVUK || {};
+/* global $ */
```

Comment:
I believe you can do something like `/* eslint-env jquery */` which may be more explicit